### PR TITLE
laravel 6 support

### PR DESCRIPTION
**Pull request by it-can**
based on https://github.com/lazychaser/laravel-nestedset/pull/399, but without the PK change

### CODE REVIEW

**Comment by jnbn** [comment]
Hopefully this'll be merged asap 😁

**Comment by dukenst2006** [comment]
Please accept pull request, can't upgrade my app to 6.0 version, getting stuck there, i have this error: `Trait 'Kalnoy\Nestedset\NodeTrait' not found`

**Comment by jnbn** [comment]
@lazychaser can you please accept it?

File: composer.json [hunk 1]
```diff
@@ -12,10 +12,10 @@
     ],
 
     "require": {
-        "php": ">=7.1.3",
-        "illuminate/support": "~5.7.0|~5.8.0",
-        "illuminate/database": "~5.7.0|~5.8.0",
-        "illuminate/events": "~5.7.0|~5.8.0"
+        "php": ">=7.2",
```
Comment (antonkomarev) [inline composer.json:15]: This is Breaking Change! There is no need to increment PHP version. `>=7.1.3` is totally fine.

**Review by antonkomarev — CHANGES_REQUESTED** [review]

File: .travis.yml [hunk 2]
```diff
@@ -1,12 +1,11 @@
...
-before_script:
-  - curl -s http://getcomposer.org/installer | php
-  - php composer.phar install --dev
-
-script: phpunit
\ No newline at end of file
+language: php
+
+php:
+  - 7.2
```
Comment (antonkomarev) [inline .travis.yml:4]: `- 7.1` could be added too

**Pull request closed without merging**